cmd/xl-v1-object.go

```diff
@@ -252,13 +259,14 @@ func (xl xlObjects) getObject(bucket, object string, startOffset int64, length i
 	}
 
 	// Get last part index to read given length.
-	lastPartIndex, _, err := xlMeta.ObjectToPartOffset(endOffset)
+	lastPartIndex, _, err := xlMeta.ObjectToPartOffset(ctx, endOffset)
 	if err != nil {
-		return errors.Trace(InvalidRange{startOffset, length, xlMeta.Stat.Size})
+		logger.LogIf(ctx, InvalidRange{startOffset, length, xlMeta.Stat.Size})
```

Comment:
Possible duplicate logging.
`xlMeta.ObjectPartOffset` is supposed to take care or already takes care of the `InvalidRange` error condition.

Reply:
done